functions/knative/wasmedge/http-server/README.md

```diff
@@ -0,0 +1,100 @@
+# Sample Function WasmEdge ( written in Rust )
+
+## Prerequisites
```

Comment:
how about including the steps to setup crun in k8s cluster?

Reply:
I'll write a guide in https://openfunction.dev](https://openfunction.dev/